common.go

```diff
@@ -62,18 +62,22 @@ func BenchmarkResultToBenchmark(name string, b *testing.BenchmarkResult, trimPre
 }
 
 // parseBenchmarkName removes the path and suffixes from benchmark info
-// Example: BenchmarkGoDriver/transform/accumulator_factory-4 -> accumulator_factory
+// Example1: BenchmarkGoDriver/transform/accumulator_factory-4 -> accumulator_factory
+// Example2: BenchmarkGoDriver/transform/bench_accumulator_factory-4 -> accumulator_factory, where "bench_" is a trimPrefixes
 func parseBenchmarkName(name string, trimPrefixes ...string) string {
-	for _, tp := range trimPrefixes {
-		name = strings.TrimPrefix(name, tp)
+	trim := func(name string) (res string) {
+		for _, tp := range trimPrefixes {
+			name = strings.TrimPrefix(name, tp)
```

Comment:
how many prefixes do we want to trim? Just once or all of them?
For instance if I have `prefix1_prefix2_prefix3_....`? If just the first one, then we can break if we found it.

Reply:
I think it makes sense to trim it just once, thank you!
Done.